pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs

```diff
@@ -735,6 +735,15 @@ private MoleculeAccessionNumbers ReadMoleculeAccessionNumberColumns(Row row)
                 moleculeIdKeys.Add(MoleculeAccessionNumbers.TagSMILES, smiles);
             }
 
+            var keggCol = ColumnIndex(SmallMoleculeTransitionListColumnHeaders.idKEGG);
+            var kegg = NullForEmpty(row.GetCell(keggCol));
+            if (kegg != null)
+            {
+                // Should have form like C07481 but we'll accept anything for now, having no proper parser
+                kegg = kegg.Trim();

```

Comment:
I think this is redundant since "NullForEmpty" already calls "Trim()".

Reply:
I don't think it does, actually, but maybe it should. There are other implementations in our code that do.

Reply:
If it were true that "NullForEmpty" does not do a "Trim()", then you would need to call "NullForEmpty" again after you do the Trim(), otherwise it's possible to end up with the empty string.

Reply:
Yeah, I've gone in and tidied that up -now NullForEmpty does the Trim().